src/main/scala/org/ergoplatform/nodeView/history/storage/modifierprocessors/ToDownloadProcessor.scala

```diff
@@ -36,11 +36,12 @@ trait ToDownloadProcessor extends ScorexLogging {
   /** return true if we estimate, that our chain is synced with the network. Start downloading full blocks after that
     */
   def isHeadersChainSynced: Boolean = pruningProcessor.isHeadersChainSynced
+  def shouldDownloadBlockAtHeight(height: Int): Boolean = pruningProcessor.shouldDownloadBlockAtHeight(height)
```

Comment:
Do we need this def? Can we make it private?

Reply:
The idea was to use it instead of `minimalFullBlockHeight`